libtenzir/include/tenzir/format/syslog.hpp

```diff
@@ -328,13 +328,11 @@ struct legacy_message_parser : parser_base<legacy_message_parser> {
               };
     const auto tag_parser = +(parsers::printable - ':');
     const auto content_parser = ~(ignore(*space) >> *parsers::printable);
-    // clang-format off
-    const auto p = ~(priority_parser >> wsignore)
-                   >> timestamp_parser >> wsignore    // timestamp
-                   >> word >> wsignore                // host
-                   >> tag_parser >> ':'               // tag
-                   >> content_parser;                 // content
-    // clang-format on
+    const auto p = ~(priority_parser >> ignore(*space)) // priority
```

Comment:
Nit (stylistic): Whitespaces can be assumed to be `ignore`d, I would rename `wsignore` to just `ws` and create a `maybe_ws` for this one.

Reply:
`ws` is used without `ignore` (on purpose) above, in `timestamp_parser`. Leaving as-is, at least for now.